clamav-rest/healthcheck.sh

```diff
@@ -1,11 +1,11 @@
-#!/bin/bash
+#!/bin/sh
 
 STATUSCODE=$(curl -L --silent --output /dev/null --write-out "%{http_code}" 127.0.0.1:8080)
 
 if test $STATUSCODE -ne 200; then
     echo "FAILURE"
     exit 1
-else 
+else
```

Comment:
Can forgo the `else` given the above will terminate .. and go with `fi` instead

Reply:
I think there's a lot of that inefficiency, I'll go through all the scripts and update.